testsuite/docker_runner/02_setup_network.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+docker network create uyuni-network-1
```

Comment:
Could we replace `docker` with a variable defaulting to `docker`? I suspect most of this could work with `podman` value out of the box

Reply:
We could but I think it does not make sense.

We do not plan to support docker and podman, it would make our life more complicated every time we change any docker call, because we would have to test it with podman.

Given we do not plan to support other runtimes than docker, there is no point to "generalize" here.

Reply:
well, it means developers have to have `docker` on their machine even though they also use `podman` for other things. Since both have basically the same commands it could have been easy. But of course can also be tackled later. I would seriously like to kick `docker` away from my laptop

Reply:
I added an issue in our project so we can tackle this later. https://github.com/SUSE/spacewalk/issues/20877